features/basics/functype.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Fetching contracted function type
+
+  You can use `functype(name)` method for that:
+
+  ```ruby
+  functype(:add) # => "add :: Num, Num => Num"
+  ```
+
+  Background:
+    Given a file named "example.rb" with:
+    """ruby
+    require "contracts"
+    C = Contracts
+
+    class Example
+      include Contracts::Core
+
+      Contract C::Num, C::Num => C::Num
+      def add(a, b)
+        a + b
+      end
+
+      Contract String => String
+      def self.greeting(name)
+        "Hello, #{name}"
+      end
+
+      class << self
+        Contract C::Num => String
+        def th(number)
```

Comment:
added commit on top of master: https://github.com/egonSchiele/contracts.ruby/commit/51cfe7cc57ef102074f52bdef06a2301c4542d6b